src/shogun/util/factory.h

```diff
@@ -85,6 +85,26 @@ namespace shogun
 		return result;
 	}
 
+	CFeatures* string_features(
+	    CFile* file, EAlphabet alpha = DNA,
+	    EPrimitiveType primitive_type = PT_CHAR)
+	{
+		REQUIRE(file, "No file provided.\n");
+		CFeatures* result = nullptr;
+
+		switch (primitive_type)
+		{
+		case PT_CHAR:
+			result = new CStringFeatures<char>(file, alpha);
```

Comment:
wasnt the idea that we dont want a `string_features` factory, buy instead somehow deal with this inside the `features` factory?

Reply:
this is something that could be done in the factory, or? (taling a sneak preview)